main.go

```diff
@@ -17,8 +17,13 @@ func main() {
 	godotenv.Load()
 
 	host := os.Getenv("CHAT_HOST")
+	// Don't disregard the error returned by Atoi here
```

Comment:
wouldn't that just throw on `net.Listen` with an invalid port?

Reply:
True - I guess the whole conversion thing was a bit contrived to let you learn about the `strconv` so this doesn't matter :)